.circleci/config.yml

```diff
@@ -605,7 +579,7 @@ jobs:
       - install_extension:
           lib_curl_command: << parameters.lib_curl_command >>
       - <<: *STEP_COMPOSER_CACHE_RESTORE
-      - <<: *STEP_COMPOSER_SELF_UPDATE
+      # - <<: *STEP_COMPOSER_SELF_UPDATE
```

Comment:
Will remove this when I remove the `_tmp` suffixes

Reply:
addressed https://github.com/DataDog/dd-trace-php/commit/165a7f057f3d17fde596964b75758faa6e0cbcea